src/mixins/common-mixin.ts

```diff
@@ -116,6 +116,12 @@ export class CommonMixin extends Vue {
       nr?.legalType === EntityType.GP)
   }
 
+  /** Returns true if the specified NR is for alteration (conversion). */
+  isSupportedAlteration (type: any): boolean {
```

Comment:
The other functions here accept a "nr" with unknown type, but here you should know the type of "type" so please add it.

Or change this function to accept a "nr" like the others.

Reply:
Thanks for the type! Resolving.